Detections/MultipleDataSources/AADAWSConsoleCorrelation.yaml

```diff
@@ -0,0 +1,40 @@
+id: 643c2025-9604-47c5-833f-7b4b9378a1f5
+name: IP with multiple failed AAD logins successfully logs in to the AWS Console.
+description: |
+	This query looks for IP addresses with multiple failed login attempts to AAD 
+	with a successful AWS Console login within the same timeframe.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+     - SigninLogs
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+  
+query: |
+
+	let signin_threshold = 5; 
+	let suspicious_signins = 
+	SigninLogs
+	| where TimeGenerated >= ago(1d)
+	| where ResultType !in ("0", "50125", "50140")
+	| summarize count() by IPAddress
```

Comment:
Since we are dealing with IP's, do we want to exclude 127.0.0.1.  Applies to all 4 queries here.

Reply:
Good suggestion, I had done this in development when also trying to bring in IPs in security alerts (which in the end was too noisy) but hadn't seen too many issues with sign ins in the data I had but definitely worth excluding still